qlib/Mapper.qm

```diff
@@ -1056,7 +1080,7 @@ Mapper mapv(DataMap);
                     return ch;
                 };
                 try {
-                    h = ah(h, v);
+                    h = ah(h, v); # FIXME I don't understand ostruct, should be "v = ah(v)", or not?
```

Comment:
no, `h = ah(h, v)` is correct, `struct` creates structured output; this is a recursive call